_docs/getting-started/windows.md

```diff
@@ -79,10 +79,13 @@ Open up a Developer Command Prompt, find an appropriate place to clone the repo,
 git clone --recursive https://github.com/pytorch/pytorch.git
 ```
 
-Run the `Developer Command Prompt for VS 2017`, browse to the repo's `\caffe2\scripts` folder and execute:
+If you want to build Python bindings, you need to edit '\scripts\build_windows.bat' and set BUILD_PYTHON to ON
 
-1. `build_host_protoc.bat`, which will build protobuf from source for your system.
-2. `build_windows.bat`, which will build Caffe2.
+Then run the `Developer Command Prompt for VS 2017`, browse to the repo's `\scripts` folder and execute:
+
+```
```

Comment:
This looks great! @pjh5 is there any case where we'd need to still run `build_host_protoc.bat`? Looks like @prasanthpul didn't need to.

Reply:
build_windows calls build_host_protoc if it hasn't already been called